src/Core/Grid/Definition/Factory/ProductGridDefinitionFactory.php

```diff
@@ -277,6 +330,15 @@ protected function getFilters()
                 (new Filter('active', YesAndNoChoiceType::class))
                     ->setAssociatedColumn('active')
             )
+            ->add((new Filter('position', TextType::class))
```

Comment:
I believe there is a subtility with this position column It should only be visible when the list is filtered by ONE category Because in other cases the position value doesn't make sense (position is only valid in the contt of ONE category)

Reply:
Only add this filter when it makes sense The product list must be filtered by ONE category for the position value to have some meaning